### ISSUE TITLE
Whether MLM loss was used during training?

### ISSUE THREAD

**Issue by FinalFlowers** [SupstarZh/WhitenedCSE#1]

**Comment by SupstarZh** [comment]
Thanks for your interest! During unsupervised training, we did not use MLM loss

**Comment by FinalFlowers** [comment]
> Thanks for your interest! During unsupervised training, we did not use MLM loss

I find there is a parameters `--num_pos 3` in `run_unsup_example.sh`, but it is not used in the code.  Does this parameter correspond to m in Formula 7 in the paper?

**Comment by SupstarZh** [comment]
> > Thanks for your interest! During unsupervised training, we did not use MLM loss
> 
> I find there is a parameters `--num_pos 3` in `run_unsup_example.sh`, but it is not used in the code. Does this parameter correspond to m in Formula 7 in the paper?

Yes, thanks for pointing it out, the current version is experimental code for some reason, I will fix it as soon as possible！
